pkg/services/ngalert/api/authorize.go

```diff
@@ -0,0 +1,11 @@
+package api
```

Comment:
I'd rename this file to be what it does as oppose to what it is. `authorization` instead of `authorize`.